docs/source/releases/latest/t

```diff
@@ -0,0 +1,37 @@
+Version [UNKNOWN] (2024-07-25)
+******************************
+
+ * *Bug fix*: Fixed error in template_basic_plugin pyproject.toml
+ * *Documentation*: Set up repo for release notes built via brassy
+
+Bug fix
+=======
+
+Fixed error in template_basic_plugin pyproject.toml
+---------------------------------------------------
+
+Fixed a small bug in template_basic_plugin's pyproject.toml. The file was incorrectly set up after the switch to poetry and my vscode TOML highlighter caught the error with the dependency.
+
+
+::
+
+    added: 
+    deleted: 
+    modified: pyproject.toml
+    moved: 
+
+Documentation
+=============
+
+Set up repo for release notes built via brassy
+----------------------------------------------
+
+Add a ``latest`` directory and add a yaml release note.
+
+::
+
+    added: docs/source/release/latest
+    added: docs/source/release/latest/20-update-pyproject-depedencies.yaml
+    deleted: 
+    modified: 
+    moved: 
```

Comment:
@evrose54 Should this file be dleeted?

Reply:
Plese!

Reply:
_plese_. I didn't add this and was confused too

Reply:
@biosafetylvl5

Reply:
Plese 😭 Deleted now